client/src/components/CreateAuction.jsx

```diff
@@ -23,12 +25,13 @@ class CreateAuction extends React.Component {
     this.handleInputChange = this.handleInputChange.bind(this);
     this.handleCategoryChange = this.handleCategoryChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleImageInput = this.handleImageInput.bind(this);
+    this.handleImageCreate = this.handleImageCreate.bind(this);
   }
 
   handleInputChange(e) {
     const name = e.target.name;
     const value = e.target.value;
-    // console.log(`target name: ${name}\ntarget value ${value}`);
     this.setState({
       [name]: value
```

Comment:
name is a reference to the variable on line 33. I set a name property on each div, so it'll take that divs name, which in this case is the same name as the state key, and itll change that key to the entered value